.github/workflows/integration.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
      strategy:
        max-parallel: 15
        matrix:
-         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', 'pypy-3.7', 'pypy-3.8']
+         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', 'pypy-3.7', 'pypy-3.8']
```

Comment:
Let's add pypy-3.9 while here, WDYT?

Reply:
Feel free to commit, I have no objection.